src/Model/ProfileModel.php

```diff
@@ -79,7 +79,8 @@ class ProfileModel extends BaseModel
      * @throws KlaviyoException
      */
     public function __construct(array $configuration ) {
-        if (empty(array_intersect_key( $configuration, array_flip(self::$identifyAttributes) ))) {
+        $arrayResponse = array_intersect_key( $configuration, array_flip(self::$identifyAttributes) );
+        if (empty($arrayResponse)) {
```

Comment:
Is this just for readability? Can we rename that variable? Something like profileIdentityValues (I don't know if I like that one much either). This is ensuring we have at least one of email, push token, phone, id to identify a profile with